packages/@biomejs/biome/bin.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/sh
+set -eu
+
+CURRENT_DIR=$(dirname "$0")
```

Comment:
```suggestion
CURRENT_DIR="$(dirname "$0")"
```

Reply:
Other than formatting, it does not change nothing at code